extra/2-piping.js

```diff
@@ -15,27 +15,37 @@
   3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment. Assign
   the final result to the variable goodCode
 */
-
-function add() {
-
+function add(a,b) {
```

Comment:
Let's go over this question in a meeting. It seems this one proved particularly hard for everyone.